libafl_nyx/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["development-tools::testing", "emulators", "embedded", "os", "no-s
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [target.'cfg(target_os = "linux")'.dependencies]
-libnyx = {git = "https://github.com/nyx-fuzz/libnyx.git",rev = "acaf7f6"}
+libnyx = { git = "https://github.com/nyx-fuzz/libnyx.git", rev = "acaf7f6" }
```

Comment:
Should we upgrade this to the latest rev?

Reply:
I started working on using the latest libnyx revision here: https://github.com/l4yton/LibAFL/tree/libnyx_latest, but will have to do some more testing. Documentation and example fuzzers would also have to be updated since `NyxProcess` gets the config directly from the `config.ron` file in the `share_dir`.